core/adapters/http_test.go

```diff
@@ -674,6 +674,7 @@ func TestHTTP_BuildingURL(t *testing.T) {
 			ep := adapters.ExtendedPath{}
 			qp := adapters.QueryParameters{}
 			err := json.Unmarshal([]byte(test.path), &ep)
+			assert.NoError(t, err)
```

Comment:
If I wrote a linter, it would say "add a failure message to your `assert`/`require`.

Reply:
fix it as you suggestion